README.md

```diff
@@ -30,3 +30,31 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+When n > 1, $$3T(n/3) + n^5$$ is true.
+
+$$3T(n/3) + n^5$$
+
+$$=3 * 3T(n/(3*3)) + 3(n/3)^5 + n^5$$
+
+$$=3 * 3 * 3T(n/(3 * 3 * 3)) + 3 * 3 * (n/(3 * 3))^5 + 3(n/3)^5 + n^5$$
+
+$$=3 * 3 * 3T(n/(3 * 3 * 3)) + (3 * 3n^5)/((3 * 3)^5) + (3n^5)/(3^5) + n^5$$
+
+$$=3 * 3 * 3T(n/(3 * 3 * 3)) + ((3 * 3)/((3 * 3)^5) + 3/(3^5) + 1) * n^5$$
+
+$$3^i * T(n/(3^i)) + (\sum_{k=0}^{i-1} ((3^k)/(3^{5k})))*n^5$$
+
+Plugging in 1 for i, we get $$3T(n/3) + n^5$$
+
+Next we substitute i with logn
+
+$$3^{logn} * T(n/(3^{logn})) + (\sum_{k=0}^{logn-1} ((3^k)/(3^{5k})))*n^5$$
+
+$$nT(1) + n^5 * ((3^{logn - 1}/(3^{5(logn-1)})) + n^5$$
```

Comment:
You're again omitting the sum here? You can't just replace the sum with the last term.

Reply:
I think I fixed it. Hopefully I'm understanding you correctly.